101-web-app-certificate-from-key-vault/README.md

```diff
@@ -0,0 +1,44 @@
+# Deploy a Web App certificate from Key Vault secret and use it for creating SSL binding
```

Comment:
This seems like a pretty involved template and user needs to know a few concepts before doing this. Would you say this is 201 or 301 level?